### ISSUE TITLE
Is it possible to share login/token between 2 micro frontend apps

### ISSUE THREAD

**Issue by alabanu** [angular-architects/module-federation-plugin#254]
Hello, 
I am trying to share the authentication token between shell app which contains the login page (localhost:4200) to the micro app  (localhost:3000) and I would like to run the micro app as standalone app without running the shell app. Is this feasible?

**Comment by kenatwex** [comment]
yes

use library to model it for consumer

https://medium.com/shopstyle-engineering/angular-injection-token-for-sharing-environment-across-nx-libraries-applications-befb9d3c6c67

**Comment by NsdHSO** [comment]
Yes, you can, but it is not recommended, because an attacker can do this just as you take that token from another application. IMHO use an API to get that information.
